.github/workflows/e2e.jdk-versions.yaml

```diff
@@ -33,7 +33,7 @@ jobs:
     timeout-minutes: 90
     strategy:
       matrix:
-        jdk: [ 8, 9, 10, 11, 12, 13, 14, 15 ]
+        jdk: [ 8 ] # , 9, 10, 11, 12, 13, 14, 15 ] TODO: add it back
```

Comment:
This seems works. Let's try to skip 9 and 10 only?

Reply:
Let's just drop 9, and 10